ansible/vars/l10k/l10k_public.yml

```diff
@@ -0,0 +1,200 @@
+SITE_HOST: 'l10k.commcarehq.org'
+COMMTRACK_SITE_HOST: 'l10k.commcarehq.org'
+NO_WWW_SITE_HOST: 'l10k.commcarehq.org'
+deploy_env: l10k
+
+active_sites:
+  cchq_ssl: True
+  cchq_redirect: True
+  cchq_http: True
+  cchq_http_j2me: False
+  cchq_http_redirect: False
+  commtrack_ssl: False
+  commtrack_http: False
+  hq_status: False
+  tableau: False
+  icds_tableau: False
+  wiki: False
+  wiki_http: False
+  motech: False
+  motech2: False
+  riakcs: False
+
+proxy_type: nginx
+
+ansible_user_password_sha_512: "{{ secrets.ANSIBLE_USER_PASSWORD_SHA_512 }}"
+fake_ssl_cert: no
+
+NEW_RELIC_KEY: "{{ secrets.NEW_RELIC_KEY }}"
+run_newrelic_plugin_agent: False
+
+DATADOG_ENABLED: True
+DATADOG_API_KEY: "{{ secrets.DATADOG_API_KEY }}"
+DATADOG_APP_KEY: "{{ secrets.DATADOG_APP_KEY }}"
+
+elasticsearch_endpoint: '{{ groups.elasticsearch.0 }}'
+elasticsearch_memory: '4096m'
+elasticsearch_cluster_name: 'deves'
+#elasticsearch_node_name: '??'
+
+apache2_key_file: '../config/{{ deploy_env }}/ssl/l10k.commcarehq.org.key'
+apache2_cert_file: '../config/{{ deploy_env }}/ssl/l10k.commcarehq.org.crt'
+
+nginx_combined_cert_file: '../config/{{ deploy_env }}/ssl/l10k.commcarehq.org.combined.crt'
+
+keystore_file: '../config/DimagiKeyStore'
+
+backup_blobdb: False
+backup_postgres: False
+backup_es: False
+backup_couch: False
+postgres_s3: False
+blobdb_s3: False
+couch_s3: False
```

Comment:
we'll need to review these at some point. we'll likely want backups on the standby if possible